deployments/finance_data_to_s3.yaml

```diff
@@ -0,0 +1,55 @@
+###
+### A complete description of a Prefect Deployment for flow 'Finance Data Publishing'
+###
+name: Finance Data Publishing
+description: null
+version: a1804d9dd517dc9375e9b17a99cd8df2
+# The work queue that will handle this deployment's runs
+work_queue_name: default
+work_pool_name: atd-data-03
+tags: []
+parameters: {}
+schedule:
+  cron: 13 7 * * *
+  timezone: null
+  day_or: true
+is_schedule_active: true
+infra_overrides: {}
+
+###
+### DO NOT EDIT BELOW THIS LINE
+###
+flow_name: Finance Data Publishing
+manifest_path: null
+infrastructure:
+  type: process
+  env: {}
+  labels: {}
+  name: null
+  command: null
+  stream_output: true
+  working_dir: null
+  _block_document_id: 48446782-86d8-41ee-9cc6-4b68e2da24c3
+  _block_document_name: anonymous-9730c7cb-f2e7-42c7-86c0-d56f968833dc
+  _is_anonymous: true
+  block_type_slug: process
+  _block_type_slug: process
+storage:
+  repository: https://github.com/cityofaustin/atd-prefect
+  reference: ch-prefect-2-test
+  access_token: null
+  include_git_objects: true
+  _block_document_id: 386bb209-603f-42ab-b361-d07010704f11
+  _block_document_name: finance-data-wip
```

Comment:
i'm curious to hear what y'all think about managing production vs development flow data. i'm mostly worried about accumulating cruft in our deployments and and storage blocks.

Reply:
I'm thinking that whoever creates a storage block used in testing flows, should then delete that block after the flows are approved. (Or reuse the block, but in any case, not keep accumulating blocks)

Reply:
It does feel crufty to me too 😢 

Another potential solution would be to make an `atd-prefect-dev` or something S3 bucket where it will upload and store our code and use that as our catch-all storage block for all things that are not yet merged into `main`.

Reply:
I've just spent a minute fussing with the prefect 2 REST api, "documented" [here](https://app.prefect.cloud/api/docs). 

I've been able to pull a list of active blocks associated to our account & workspace, and more generally automate interacting with the prefect service. What would you all think about the idea of writing a prefect ETL to clean up blocks that we don't put on a "save-forever" list after, say, 60 days?

An automatic clean-up robot as it were?

Reply:
Those are cool ideas, Frank and Charlie!  I mentioned this in dev sync today, but I was wondering if there would be a downside to creating both production and staging/development blocks per flow. That way someone working on a flow could update the reference in the staging block and go from there. I might have overlooked something, though. Most of our development on flows seems to be one person working on one flow at a time but that could be a limitation.

I've been thinking about this since the service bot flows have config for test/dev/staging and production. I created staging and production secret blocks to see how that goes.